### ISSUE TITLE
Are we forking the FT o-colors component or creating our own system?

### ISSUE THREAD

**Issue by parkerpearson** [Pearson-Higher-Ed/o-colors#2]
Specifically, are we going to have the concept of use cases or simply stick with a palette of colors?

**Comment by aarmour** [comment]
I'm going to dig into FT's module. We won't fork it, but it should provide some inspiration.

**Comment by aarmour** [comment]
Added issue #3 to discuss v1 scope for this component.

**Comment by parkerpearson** [comment]
Sounds good. My 2¢: at first it seemed like FT module was a little over-wrought, but after considering it I like the idea of use-cases because I can see us using the same color in two semantically different ways.

**Comment by aarmour** [comment]
I agree, but I'd like to start with the simplest thing that works and then iterate when we see opportunities to provide more value. Maybe we can start by generating a list of the potential semantic color names that would serve as the base for all dependent components. Dependents would derive their colors from the base colors. [Example](http://sassmeister.com/gist/924a489b149f6c2d0b72).

**Comment by aarmour** [comment]
Some additional references:

* [GitHub Primer](http://primercss.io/colors/)
* [Atlassian](https://design.atlassian.com/2.2/product/foundations/colors/)
* [Bootstrap](http://getbootstrap.com/css/#less-variables-colors)
* [Foundation](http://foundation.zurb.com/docs/components/global.html)
* [Material Design](https://www.google.com/design/spec/style/color.html)
